packages/next-app/public/tokens/logoUri.ts

```diff
@@ -0,0 +1,23 @@
+interface ITokenLogoUri {
+  [key: string]: string;
+}
```

Comment:
Do we really need to define a special type for this? I'd argue that not unless you plan to be extending the object.